src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_Dynamic.cs

```diff
@@ -120,37 +116,32 @@ internal struct DynamicAttributeValidator
             private readonly NamedTypeSymbol _synthesizedMyDelegateType;
             private bool[] _expectedTransformFlags;
 
-            private DynamicAttributeValidator(CSharpCompilation compilation)
+            private DynamicAttributeValidator(ModuleSymbol module)
             {
-                _comp = compilation;
-                _srcAssembly = compilation.SourceAssembly;
-                NamespaceSymbol globalNamespace = _srcAssembly.Modules[0].GlobalNamespace;
```

Comment:
>globalNamespace [](start = 32, length = 15)

It feels like we could keep this local (with adjusted initializer) and avoid many unnecessary changes below.